Tests/MapboxMapsTests/Foundation/Camera/MockMapboxMap.swift

```diff
@@ -46,8 +47,8 @@ final class MockMapboxMap: MapboxMapProtocol {
         var from: CGPoint
         var to: CGPoint
     }
-    let dragCameraOptionsStub = Stub<DragCameraOptionsParams, CameraOptions>(defaultReturnValue: CameraOptions())
-    func dragCameraOptions(from: CGPoint, to: CGPoint) -> CameraOptions {
+    let dragCameraOptionsStub = Stub<DragCameraOptionsParams, MapboxMaps.CameraOptions>(defaultReturnValue: CameraOptions())
+    func dragCameraOptions(from: CGPoint, to: CGPoint) -> MapboxMaps.CameraOptions {
```

Comment:
Curious why these prefixes needed to be added

Reply:
After I imported `MapboxCoreMaps_Private` it had some clashes.